app/views/dashboards/_sublist.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class="sublist"></div>
 <div class="loading" id="loading-articles">
-  <%= t("dashboard.loading") %>
+  <%= t("core.loading") %>
```

Comment:
This file has already been tokenized

Reply:
Ok, do you have any suggestions for the fix?
Should I revert this file back and keep the other changes or is there anything else I can improve?

Reply:
You can keep the other changes, of course, just remove the ones in this file.